starters/features/partytown/src/components/partytown/partytown.tsx

```diff
@@ -0,0 +1,36 @@
+import { partytownSnippet, PartytownConfig } from '@builder.io/partytown/integration';
+
+/**
+ * Props for `<QwikPartytown/>`, which extends the Partytown Config.
+ *
+ * https://github.com/BuilderIO/partytown#config
+ *
+ * @public
+ */
+export interface PartytownProps extends PartytownConfig {}
+
+/**
+ * @public
+ * You can pass setting with props
+ */
+export const QwikPartytown = (props: PartytownProps): any => {
+  if (typeof document !== 'undefined' && !document._partytown) {
```

Comment:
not a fan of this, check it will not work in SSR which is the main thing in Qwik, i would remove this check completely it should be up to the developer to not have two partytowns in the same page